hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/ReconContainerMetadataManagerImpl.java

```diff
@@ -110,25 +94,9 @@ public void close() throws Exception {
   public void initNewContainerDB(Map<ContainerKeyPrefix, Integer>
```

Comment:
TODO: Rename this

Reply:
Followed Aravindan's suggestion: `reinitWithNewContainerDataFromOm`